di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/config/credentialissuer/CredentialIssuer.java

```diff
@@ -13,4 +13,5 @@ public record CredentialIssuer(
         boolean sendOAuthJAR,
         boolean sendEncryptedOAuthJAR,
         String expectedAlgo,
-        String userInfoRequestMethod) {}
+        String userInfoRequestMethod,
+        String publicEncryptionJwkBase64) {}
```

Comment:
each CRI configured has its own public key JWK that corresponds to the new KMS key configured per stack/account

Reply:
Food for thought: ^ should we document the above in the read me